### PR TITLE
chore(cleanup): drop proptypes usage

### PR DESCRIPTION
**Pull request by arpl**
# Purpose of PR
Drop PropTypes dependency since we don't support them. Update all `yarn.lock`s after issuing `lerna clean && lerna bootstrap` from project root to keep them aligned.

### CODE REVIEW

File: packages/components/forma-36-react-datepicker/yarn.lock [hunk 1]
```diff
@@ -734,20 +734,6 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@contentful/forma-36-react-components@^3.25.1":
```
Comment (m10l) [inline packages/components/forma-36-react-datepicker/yarn.lock]: I'm not sure if this should be removed

File: packages/components/forma-36-react-timepicker/yarn.lock [hunk 2]
```diff
@@ -734,25 +734,6 @@
     exec-sh "^0.3.2"
     minimist "^1.2.0"
 
-"@contentful/forma-36-react-components@^3.25.1":
```
Comment (m10l) [inline packages/components/forma-36-react-timepicker/yarn.lock]: Same, not sure this should have been removed. Maybe try running from the root

File: packages/forma-36-react-components/yarn.lock [hunk 3]
```diff
@@ -1405,22 +1405,10 @@
     puppeteer "^1.2.0"
     yargs "^11.1.0"
 
-"@contentful/forma-36-fcss@^0.0.34":
```
Comment (m10l) [inline packages/forma-36-react-components/yarn.lock]: Same here

File: packages/forma-36-react-components/yarn.lock [hunk 4]
```diff
@@ -1405,22 +1405,10 @@
...
-  resolved "https://registry.yarnpkg.com/@contentful/forma-36-fcss/-/forma-36-fcss-0.0.34.tgz#afc4514e9b946c67339202ac61df94dc702b3bbe"
-  integrity sha512-AhK2QJn2pQ/MWR8d10H4B6g625MU1rz4lkakkw8/0E0KFpeow53XW2lbN0vTdOXDaNM1TeMOP0F700qFIoeMkA==
-  dependencies:
-    "@contentful/forma-36-tokens" "^0.5.1"
-
 "@contentful/forma-36-tokens@0.2.0":
   version "0.2.0"
   resolved "https://registry.yarnpkg.com/@contentful/forma-36-tokens/-/forma-36-tokens-0.2.0.tgz#c4a1062e7acc2ae0ae5699e7c8606d5b5047a1c5"
 
-"@contentful/forma-36-tokens@^0.5.1":
```
Comment (m10l) [inline packages/forma-36-react-components/yarn.lock]: And here

File: packages/components/forma-36-react-datepicker/yarn.lock (hunk 1 above)
Comment (arpl) [reply packages/components/forma-36-react-datepicker/yarn.lock]: Running `lerna bootstrap` from root will link the locally found packages hence the difference between the `lock` files. We should commit these locks and we won't see differences when pulling and `lerna bootstrap`-ing.

**Pull request closed without merging**